Packages/com.unity.inputsystem/InputSystem/Editor/Internal/EditorHelpers.cs

```diff
@@ -41,6 +41,16 @@ public static string GetTooltip(this SerializedProperty property)
             return string.Empty;
         }
 
+        public static string GetHyperlink(string text, string path)
+        {
+            return "<a href=\"" + path + $">{text}</a>";
```

Comment:
Am I hallucinating or you're missing the second `\"`?

Reply:
Fixed in https://github.com/Unity-Technologies/InputSystem/pull/1903/commits/2394e9cd32375cbd063b5e00bf37e808e52f3c28